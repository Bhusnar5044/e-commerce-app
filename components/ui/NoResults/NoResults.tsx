import { memo } from 'react';

export const NoResults = memo(() => {
    return <div className="flex items-center justify-center h-full w-full text-neutral-500">No results found.</div>;
});

NoResults.displayName = 'NoResults';
