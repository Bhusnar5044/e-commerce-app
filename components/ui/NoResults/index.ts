export { NoResults as default } from './NoResults';
