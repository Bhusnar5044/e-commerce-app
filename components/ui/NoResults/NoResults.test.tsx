import { render } from '@test-utils';
import { NoResults } from './NoResults';

describe('NoResults', () => {
    it('should render properly', () => {
        const { container } = render(<NoResults />);
        expect(container).toMatchSnapshot();
    });
});
