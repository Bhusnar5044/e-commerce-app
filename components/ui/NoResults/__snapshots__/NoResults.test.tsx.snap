// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoResults should render properly 1`] = `
<div>
  <div
    class="flex items-center justify-center h-full w-full text-neutral-500"
  >
    No results found.
  </div>
</div>
`;
